### ISSUE TITLE
Contact pencil icon doesn't show the full contact cards

### ISSUE THREAD

**Issue by laspic** [protz/thunderbird-conversations#1402]
* **Thunderbird version:** 68.2.2
* **Thunderbird Conversation version:** 2.15.16
* **Operating System:** Xubuntu 18.04

**What did you do?**
I try to look a contact card directly from the small popup on the name of a participant of a conversation, in clicking on the + button.

**What did you expect to happen?**
I expect the contact card to open and display all data recorded for the target contact.

**What actually happened?**
The contact card open and display not all data recorded for the target contact but only some of them, namely email address and phone number. The name and surname, photo field are particularly empty.
In other words, the contact card displays a different content depending on whether it is opened from Thunderbird Conversation add-on or the default user interface.

**Comment by Standard8** [comment]
If you're seeing the "+" button after you've clicked the down-arrow expansion button, then that means Conversations hasn't found the card in your address book.

If it is a pencil icon, then it has found the card.

Are you sure the contact is listed in your address book and is using exactly the same email address?

I've just double-checked and it is working fine here for me. You might want to also try looking at the Error Console (Tools -> Developer Tools) to see if there's any errors being displayed when you're opening messages or hovering over contacts.

**Comment by laspic** [comment]
I was wrong about the "+" button, it is a pencil icon indeed. My mistake.

Otherwise, I confirm that something doesn't work well. All the contacts are listed in my address book (Personal Addresses or Collected Addresses) and the same email addresses are exactly the same.

I also tried to open a contact card with the pencil icon to modify some fields. Re-opening the card after saving still shows empty fields (doing the same operation with the default UI is fine). Sometimes even the email address is not kept, the pencil icon opens a very virgin card.

Here is what the Error Console shows. 
− If I want to open an existing card, but it opens a virgin one:
`TypeError: jbCatMan.getReducedCategoriesForHierarchyMode is not a functioncommon-card-overlay.js:201:36`
− If I want to save modifications on an existing card, but it saves nothing: no message.

**Comment by laspic** [comment]
The error message gave me an idea. And if the cited function 'jbCatMan.getReducedCategoriesForHierarchyMode' could be a part of the [CategoryManager](https://github.com/jobisoft/CategoryManager/) add-on ?

I tried to enable it... and everything magically falls into place!

Problem solved! Is there a chance to see that incompatibility fixed?

**Comment by Standard8** [comment]
You'll need to log an issue with the CategoryManager author - as far as I can tell, Conversations is using the standard methods that Thunderbird uses for opening the contact dialogs.

I suspect the issue is the way in which the CategoryManager is attempting to load its category tools, but they'll know more about how that is working.

Closing as this isn't directly an issue with Conversations itself.
